docs/tables/base.py

```diff
@@ -12,29 +13,66 @@
 
 
 class BaseTabler(abc.ABC):
+    """Base class for tablers.
+
+    Attributes:
+        alignment (list[str] | None): Optional alignment of table columns.
+    """
+    # Class attributes
+    alignment: list[str] | None = None
```

Comment:
If this is a class variable (as opposed to instance variable) the type should be wrapped in `ClassVar[...]`

Reply:
No requirement for this, should only be utilised if class variables should not be set on an instance. The comment indicates that the attributes available at the class level are declared below, not that they are exclusively set by a class.